src/http_link.c

```diff
@@ -150,14 +154,28 @@ int http_link_write(uv_link_t *link, uv_link_t *source, const uv_buf_t bufs[], u
                         response->body_size, &resp, &resp_size);
         }
       }
-      else
+      else // Gzip initialized
       {
-        context->request.enable_compression = false;
+        compress_data(response, resp, nread, &resp, &resp_size);
       }
     }
-    else // Gzip initialized
+
+    if (context->first_response)
     {
-      compress_data(response, resp, nread, &resp, &resp_size);
+      context->first_response = false;
+      double timeDiff = (uv_hrtime() - context->request_time) / 1000000.0;
+      struct tm *timeinfo;
+      timeinfo = localtime(&context->request_timestamp);
+      char timeString[256];
+      strftime(timeString, sizeof(timeString), "%d/%b/%Y:%T %z", timeinfo);
+      // IP - [response time] - [date time] "GET url http" HTTP_STATUS_NUM host
+      log_debug("%s - [%.3fms] - [%s] \"%s\" %u %s",
```

Comment:
please add here `[%.5ms]` to be sure to have enough space for response time.

Reply:
Number after dot is used for decimal points. 
> For a, A, e, E, f and F specifiers: this is the number of digits to be printed after the decimal point (by default, this is 6).
> http://www.cplusplus.com/reference/cstdio/printf/

Reply:
sorry I mixed that with `%-5s`, just like its done here: https://github.com/bleenco/bproxy/blob/master/src/log.c#L66